shared/common-adapters/plain-input.js.flow

```diff
@@ -0,0 +1,86 @@
+// @flow
+import {Component} from 'react'
```

Comment:
I'm not sure if duplicating this code is a good idea -- the existing input component is pretty fragile with respect to dealing with controlled vs. uncontrolled states, and the new one changes some behavior, so future changes would have to update everything. Perhaps it's possible to make the existing input component into a tiny wrapper that adds the containing boxes etc. to a core input component, that's shared with `NewInput`?